src/config.cr

```diff
@@ -11,6 +11,8 @@ class Config
   property session_secret : String = "mango-session-secret"
   property library_path : String = File.expand_path "~/mango/library",
     home: true
+  property library_cache_path = File.expand_path "~/mango/library.yml.cbz",
```

Comment:
Here is a typo. it should be `~/mango/library.yml.gz`

Reply:
lol that's a terrible typo. Thanks for the heads-up!